spec/fixtures/hiera/hathitrust.yaml

```diff
@@ -84,6 +85,11 @@ nebula::virtual::users::all_users:
     comment: Solr user
     uid: 123458
     home: /var/lib/solr
+  prometheus:
+    uid: 108
+    gid: prometheus
```

Comment:
Adding Prometheus users/groups to the HathiTrust hiera is really just adding more technical debt to nebula, but I couldn't think of a better way to handle this (since hathitrust.yaml overrides this hash instead of merging, which is also our desired behavior for this).

I think the way to actually clean this up is (as with many other problems) to have more than one puppet module so that all these unrelated profiles aren't stepping on each other's toes.